packages/@aws-cdk/aws-fsx/test/lustre-file-system.test.ts

```diff
@@ -465,7 +465,67 @@ describe('FSx for Lustre File System', () => {
             vpc,
             vpcSubnet,
           });
-        }).toThrowError('perUnitStorageThroughput can only be set for the PERSISTENT_1 deployment type');
+        }).toThrowError('perUnitStorageThroughput can only be set for the PERSISTENT_1/PERSISTENT_2 deployment types');
+      });
+    });
+
+    describe('perUnitStorageThroughput_Persistent_2', () => {
+      test.each([
+        125,
+        250,
+        500,
+        1000,
+      ])('valid perUnitStorageThroughput of %d', (throughput: number) => {
+        lustreConfiguration = {
+          deploymentType: LustreDeploymentType.PERSISTENT_2,
+          perUnitStorageThroughput: throughput,
+        };
+
+        new LustreFileSystem(stack, 'FsxFileSystem', {
+          lustreConfiguration,
+          storageCapacityGiB: storageCapacity,
+          vpc,
+          vpcSubnet,
+        });
+
+        Template.fromStack(stack).hasResourceProperties('AWS::FSx::FileSystem', {
+          LustreConfiguration: {
+            DeploymentType: LustreDeploymentType.PERSISTENT_2,
+            PerUnitStorageThroughput: throughput,
+          },
+        });
+      });
+
+      test('invalid perUnitStorageThroughput', () => {
+        lustreConfiguration = {
+          deploymentType: LustreDeploymentType.PERSISTENT_2,
+          perUnitStorageThroughput: 1,
+        };
+
+        expect(() => {
+          new LustreFileSystem(stack, 'FsxFileSystem', {
+            lustreConfiguration,
+            storageCapacityGiB: storageCapacity,
+            vpc,
+            vpcSubnet,
+          });
+        }).toThrowError('perUnitStorageThroughput must be 125, 250, 500 or 1000 MB/s/TiB');
+      });
```

Comment:
I would probably make this test also parametrized, so that it covers more scenarios than just `1`, like the tests for invalid `storageCapacity` below.

Reply:
Agreed. Updated as part of Revision 4